.metrics.json

```diff
@@ -32,7 +32,7 @@
       }
     },
     {
-      "name":        "cv32e40p_ci_check_regression",
+      "name":        "cv32e40p_ci_check_dev",
```

Comment:
This name should be the default `name_of_ci_check_regression` used by `ci_check`.